Dockerfile.release

```diff
@@ -6,7 +6,7 @@ WORKDIR /parallel
 
 COPY . /parallel
 
-RUN cargo build --$PROFILE --bin parallel-dev
+RUN cargo build --$PROFILE --bin parallel-dev --features runtime-benchmarks
```

Comment:
This feature should only be used in benchmark, not the online binary.

Reply:
it'll make benchmark on our server easier I think